spec.html

```diff
@@ -35031,11 +35031,13 @@ <h1>IsDetachedBuffer ( _arrayBuffer_ )</h1>
 
       <!-- es6num="24.1.1.3" -->
       <emu-clause id="sec-detacharraybuffer" aoid="DetachArrayBuffer">
-        <h1>DetachArrayBuffer ( _arrayBuffer_ )</h1>
-        <p>The abstract operation DetachArrayBuffer with argument _arrayBuffer_ performs the following steps:</p>
+        <h1>DetachArrayBuffer ( _arrayBuffer_ [ , _owner_ ] )</h1>
+        <p>The abstract operation DetachArrayBuffer with argument _arrayBuffer_ and optional argument _owner_ performs the following steps:</p>
         <emu-alg>
-          1. Assert: Type(_arrayBuffer_) is Object and it has [[ArrayBufferData]] and [[ArrayBufferByteLength]] internal slots.
+          1. Assert: Type(_arrayBuffer_) is Object and it has [[ArrayBufferData]], [[ArrayBufferByteLength]], and [[ArrayBufferOwner]] internal slots.
           1. Assert: IsSharedArrayBuffer(_arrayBuffer_) is *false*.
+          1. If _owner_ is not provided, let _owner_ be *undefined*.
+          1. If _arrayBuffer_.[[ArrayBufferOwner]] is not _owner_, throw a *TypeError* exception.
```

Comment:
What kind of value is this? “Is not” seems very vague. It’d be ideal to specify an algorithm for comparing these values (if the type of them can’t be tightly specified), like SameValue.

Reply:
The owner will always come from some other specification, and be a constant. I'd be fine to switch to SameValue, but I don't see what it adds, what ambiguity it resolves.

Reply:
It reduces how much context i need to read another spec to have.

Reply:
The specification already uses is and SameValue fairly interchangeably. I'd suggest a separate issue for that (although I kind of remember there being one already).

Reply:
In cases where the types are known, yes, but they’re not specified here.

I see from the other PR that it’s a string; of this PR asserts somehow that the slot is a string, then I’m comfortable with “is” phrasing.